src/App/App.js

```diff
@@ -138,12 +138,17 @@ export class App extends Component{
     // if (!name) {
     //   return false
     // }
-    const promise = await fetcher.getMovieComments(movieId) 
-    console.log(promise)
+    const promise = await fetcher.getMovieComments(movieId)
+    if (promise === 'Failed to Fetch'){
+      this.setState({error: promise})
+    } else {
```

Comment:
I saw the use of `new Error(error)` elsewhere. Would you want to implement this class later if we've got the chance?